xml/obs_ag_overview_filesystem.xml

```diff
@@ -647,13 +647,58 @@ auto_cleanup_after_days: 30
          <para>see Reference Guide for more information</para>
         </entry>
        </row>
+       <row>
+        <entry align="left" valign="top">
+         <para>kiwi_image_editor</para>
+        </entry>
+        <entry align="left" valign="top">
+         <para>enable/disable Kiwi Image Editor</para>
+        </entry>
+        <entry align="left" valign="top">
+         <para>true <literal>false</literal></para>
+        </entry>
+        <entry align="left" valign="top">
+         <para></para>
+        </entry>
+       </row>
+       <row>
+        <entry align="left" valign="top">
+         <para>cloud_upload</para>
+        </entry>
+        <entry align="left" valign="top">
+         <para>enable/disable Cloud Upload setup</para>
+        </entry>
+        <entry align="left" valign="top">
+         <para>true <literal>false</literal></para>
+        </entry>
+        <entry align="left" valign="top">
+         <para></para>
+        </entry>
+       </row>
       </tbody>
      </tgroup>
     </table>
     <para>Example feature.yml</para>
     <screen>
-features:
-  image_templates: true
+<![CDATA[
```

Comment:
Do we need to include the feature flags here? I guess we will remove the feature flag with the release!

Reply:
Then whenever it is going to be removed this docu should change. Today and now we have a feature flag that controls it.

Reply:
I actually don't see the value of that. It is duplicate work or we might even forget to update. The Unstable branch is not supported. In the official released version there should no feature flag at all.

Reply:
So, this docu is for the releases? What happen if anybody installs that and is not working because the flag (the default value is disabled).... Does it make sense to have this info until it gets removed?

Reply:
Yes, this documentation is for releases. Releases should not contain feature flags if you ask me.

Reply:
Ok, I thought that this feature in particular is something the admin can disable to not show this in the web UI if he don't want to provide it. For me is something optional to an OBS instance, so the flag make sense for me.

Reply:
To quote PO: "The feature should be actively promoted by not hiding it" -> so that means we intentionally don't want to hide it! Also it does not harm if it is not hidden, the only thing what would happen is that the user gets a flash message with 'No cloud backend configured, please contact the administrator'

Reply:
Nice quote and I don't agree with... 
Promoting by disturbing end users with an non configured feature seems a little bit strange to me 😈
If the admin won't use this feature never... he will keep "promoting" the feature for the end users, always! Non sense.

Reply:
Well, then you should address this to PO 😄 
Anyway, doesn't change anything because if you ask me, feature flags are something for development and not for administrators. And I would prefer to remove all feature flags before we release a new version.

Reply:
Ok, @adrianschroeter / @mlschroe what do you think about it?

Reply:
Feature flags are of course something for administrators. Why do I need them in development?

Reply:
https://martinfowler.com/articles/feature-toggles.html

To ship code early and fast without interrupting users. We never hided finished features from users / administrators in the past.

Reply:
In development for trying it out I guess...

Reply:
>  We never hided finished features from users / administrators in the past.

By **"We"** you mean the build.opensuse.org admins... but in general it could be helpful for other admins of other instances to not show certain features the won't use.

Reply:
We can not  hide features from admins as they control feature.yml.
We, as admins of the reference server, have hidden features from (*certain*) users.

Reply:
> We can not hide features from admins as they control feature.yml.
We actually can by removing the feature switches from the source code (what we actually did)

> As you see in this section, that Moises extended, we even point admins to feature toggles.
 Write

Which was pointless as we removed the features switches from the source code so changing the feature.yml had no effect.

Reply:
So far we used feature switches only to hide new features from users before the official release. After the official release we removed the features checks from the source code. If we want to change that in the future we need to talk about it but so far we only used it for this purpose (which I call as development)

Reply:
I tend to agree with Christian in the way that we have already the /configuration route to enable or hide features. Having a second place makes it not easier.

In any case we need to avoid to ask admins to manually enable features after update which should be there by default. means our built-in defaults must fit.

Reply:
I think we all agree in that maybe we need to use the feature flags only for trying out things for a certain users, but to release a possible optional feature we need to convert that flag into a configuration entry. I mean by moving the calculation of the "flag" from the feature flag to a thing like CONFIG["flag"] that way other admins can disable features for production.

Reply:
Nevertheless I will keep this until things change. Maybe worth to have a card in our backlog for doing this. What do you think? @hennevogel @ChrisBr

Reply:
Fine for me

Reply:
> After the official release we removed the features checks from the source code. 

Sorry this is simply not true. Look at the history of [config/feature.yml](https://github.com/openSUSE/open-build-service/commits/master/src/api/config/feature.yml). All of those switches are still there in the code.

> I tend to agree with Christian in the way that we have already the /configuration route to enable or hide features.

What feature (in the sense of **features** we talk about here, not difference behavior for certain things) can you configure from inside the app? None.

> In any case we need to avoid to ask admins to manually enable features after update which should be there by default.

I really would do it the other way around. Enabling those far reaching features that we have behind a feature flag should not *self-enable* all of the sudden. They have implications. Image templates have to be there, cloud shit has to be configured etc.  It's up to the admin to make sure these prerequisites are there.

We ensure this BTW currently by marking config/features.yml as `%config(noreplace)` in our [spec file](https://github.com/openSUSE/open-build-service/blob/master/dist/obs-server.spec#L599) and features that are not inside the yml file will be considered off.

> So, to release a possible optional feature we need to convert that flag into a configuration entry.

I'm not sure I agree. Of course you can do that and make everything runtime configurable, from inside the app but then you don't need feature flags at all.

Also not everything is runtime configurable...

Reply:
Beside this conversation..., do you agree with letting this go as it is right now? and make changes after, probably adding first a card in our sprint backlog.

So, this documentation will remain as it is in this PR and we will change it if we change the way we use the feature flags.